### PR TITLE
Replication script readme

### PR DESCRIPTION
**Pull request by esb7**
Documentation and updates to the usage() for the replication script.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,7 +35,7 @@ fab/bottom_reinforcement.stl : baseline/bottom_reinforcement.stl
 # Use replicate_model script to create a mk3 build plate full of bottom_reinforcement pieces
 # Prints in 7:29
 fab/bottom_reinforcement-full_plate-fast.3mf:
-	python scripts/replicate_model.py --model baseline/bottom_reinforcement-single-fast.3mf --output $@ --x_gap 1 --y_spacing 8.2 --x_offset 4 --front_offset 10
+	python3 scripts/replicate_model.py --model baseline/bottom_reinforcement-single-fast.3mf --output $@ --x_offset 4 --front_offset 10 --rear_offset 5 --x_gap 1 --y_spacing 8.2
```
Comment (hzeller) [inline Makefile]: I'd break this in multiple lines with backslashes, so that it is easier to read and modify.

**Comment by hzeller** [comment]
TBH, I have not really understood the difference between spacing and gap. Wouln't I get a bigger gap if I just add more spacing ? Or is this to better reflect the situation with 'spacing between things' vs. 'at the edge of the bed' ?

**Comment by esb7** [comment]
I'll clarify that in the readme.  Spacing allows the objects to overlap.  Gap does not.

- Eric

**Comment by hzeller** [comment]
thanks!

**Pull request merged**